src/rendering/vcPolygonModel.cpp

```diff
@@ -144,7 +145,12 @@ udResult vcPolygonModel_CreateFromRawVertexData(vcPolygonModel **ppPolygonModel,
   if (pPolygonModel->pMeshes[0].materialID == vcPMST_Count)
     UD_ERROR_SET(udR_Unsupported);
 
-  UD_ERROR_CHECK(vcMesh_Create(&pPolygonModel->pMeshes[0].pMesh, pMeshLayout, totalTypes, pVerts, pPolygonModel->pMeshes[0].numVertices, pIndices, indexCount, (pIndices == nullptr) ? vcMF_NoIndexBuffer : vcMF_IndexShort));
+  if (pIndices == nullptr)
+    meshFlags = meshFlags | vcMF_NoIndexBuffer;
+  else if (flags & vcPMF_IndexShort)
+    meshFlags = meshFlags | vcMF_IndexShort;
```

Comment:
Could auto-detect this flag (when `vertCount <= 65535`)? Then `vcPolyModelFlags` could be removed.

Thoughts?

Reply:
This flag tells us the format of `pIndices`. We're passing in our index data as a `void *` so we need a flag to let us know how to interpret this data.

Reply:
@peterAdams1014 Just to update, I removed the flags and set `vcPolygonModel_CreateFromRawVertexData()` to accept index data as `uint32_t`